lib/compat/wordpress-6.5/fonts/class-wp-font-utils.php

```diff
@@ -215,13 +215,17 @@ private static function apply_sanitizer( $value, $sanitizer ) {
 		}
 
 		/**
-		 * Provide the expected mime-type value for font files per-PHP release. Due to differences in the values returned these values differ between PHP versions.
+		 * Returns the expected mime-type values for font files, depending on PHP version.
 		 *
-		 * This is necessary until a collection of valid mime-types per-file extension can be provided to 'upload_mimes' filter.
+		 * This is needed because font mime types vary by PHP version, so checking the PHP version
+		 * is necessary until a list of valid mime-types for each file extension can be provided to
+		 * the 'upload_mimes' filter.
 		 *
 		 * @since 6.5.0
 		 *
-		 * @return Array A collection of mime types keyed by file extension.
+		 * @access private
```

Comment:
This is `private` but method uses `public`? Is that right @creativecoder?

Reply:
Yes, most of the other methods in this class are like this: they need to be public for access from other classes, but I don't think we're ready for these to be part of the public API for WordPress, similar to how some global functions are prefixed with `_` and marked `@access private`.

But let me know if there's a better way to handle this.

Reply:
TIL - thank you.

I'm not aware of any better way. Just double checking that this wasn't an oversight.